tools/Makefile.shared

```diff
@@ -378,3 +373,7 @@ depend: beforedepend
 .PHONY: clean install beforedepend depend
 
 include .depend
+
+# Local Variables:
```

Comment:
Please don't add emacs local variables to files, we use lots of different editors and lots of different emacs versions with different configurations, and for us local variables are more trouble than they're worth.